husky_control/config/control_left_ur5.yaml

```diff
@@ -0,0 +1,22 @@
+l_arm_controller:
```

Comment:
@TheDash these don't show up in the ``rosservice call /controller_manager/list_controllers`` list - might be related to #40